ui/raidboss/popup-text.ts

```diff
@@ -1648,6 +1688,20 @@ export class PopupText {
     return div;
   }
 
+  _updateCountdownInternal(span: HTMLElement, endTime: number): void {
+    const updateCountdown = () => {
+      let remainingTime = (endTime - +new Date()) / 1000;
```

Comment:
I'm not as familiar with `Date` as maybe I should be. Is this construction of `- +` to avoid odd behavior relating to it?

Reply:
It's just shorthand for `(new Date()).getTime()`, to avoid the required extra parenthesis, I believe.

This is expanded to something like:

```ts
      let remainingTime = (endTime - (0 + new Date())) / 1000;
```

Which then gets expanded to:
```ts
      let remainingTime = (endTime - (0 + (new Date()).getTime())) / 1000;
```